### ISSUE TITLE
Data Management

### ISSUE THREAD

**Issue by carschno** [LAHTeR/document_segmentation#42]
## Fixes
- [x] Ignore 'inventaris deel' if it is a number, only keep letters
- [x] TBD: mark `OUT` pages as document, except for empty pages
- [x] Introduce `END_BEGIN` label

## Necessary improvements
- [X] Implement script for downloading inventories (`download_inventories.py`)
- [x] Implement script for annotating inventories ('segment_inventories.py`)
- [x] Implement script for storing annotated data (locally); currently all inventories are stored with label `UNK`, and annotated on the in the training script.
- [x] add thumbnail to output (see #47 )

## Perhaps; needs investigation
- [ ] Mark training and validation sets (currently dataset is shuffled upon training time)
- [ ] ~~Store data in Json format on HUC server~~
- [ ] Store annotated Jsons in separate directory

**Comment by carschno** [comment]
The current format of (intermediately) storing the data on a document level works only for training the model on the basis of documents, but does not allow
- storing inventories without annotations
- storing pages outside of documents with annotations

Therefore, this should change to store the data based on inventories:
```python
Document:
    inv_nr: str
    pages: list[Page]
```

```python
Page:
    scan_nr: PositiveInt
    folio_nr: PositiveInt # Optional?
    external_ref: str
    label: Optional[Label]
    regions: list[Region]
```
The label needs to be updated to provide an Unknown (`UNK`) label. This can take index `0` so that a boolean check on the label of a page works implicitly.
